h/views/feeds.py

```diff
@@ -15,7 +17,9 @@
 
 def _annotations(request):
     """Return the annotations from the search API."""
-    result = search.Search(request, stats=request.stats).run(request.params)
+    schema = SearchParamsSchema()
```

Comment:
When I spoke with @lyzadanger about building the schemas she was keen on keeping the schema's separate. I could see a case for re-using a schema if it's exactly the same but I wonder in this case if we want to expose everything we expose in the /api/search here as well. One thing that comes to mind is _separate_replies.